packages/clerk-js/src/core/clerk.ts

```diff
@@ -1098,6 +1100,13 @@ export default class Clerk implements ClerkInterface {
         void this.handleUnauthenticated({ broadcast: false });
       }
     });
+
+    // set cookie on token update
+    eventBus.addEventListener('token:update', event => {
```

Comment:
🙃  We should consider centralizing event definitions

Reply:
like this ? 5d67113ea7e6ec42987234fac948739462da8ee4